src/main/java/com/haxwell/apps/quizki/_servletFilters/CorsFilter.java

```diff
@@ -21,7 +21,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
 		HttpServletResponse response = (HttpServletResponse) servletResponse;
 		HttpServletRequest req = (HttpServletRequest)servletRequest;
 
-		response.setHeader("Access-Control-Allow-Origin", "http://localhost:8100"); // was *
+		response.setHeader("Access-Control-Allow-Origin", "*"); // was http://localhost:8100
```

Comment:
This defines where the requests are allowed to come from.. it should be the URL of the client you are using to reach the backend. * is too general.

Reply:
Would this URL be different on other systems?

Reply:
Yes, likely so.. we're probably always going to be using Angular, so a value of `http://localhost:4200` is appropriate.